### PR TITLE
Clear incoming source

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
[Clear source from incoming elements](https://trello.com/c/MXsnfKfz/3757-clear-source-from-incoming-elements)

**Comment by amontenegro** [comment]
@wjrsimpson could you please take a look a this?

Thanks

File: orcid-core/src/main/java/org/orcid/core/adapter/impl/MapperFacadeFactory.java [hunk 1]
```diff
@@ -261,23 +261,6 @@ public void registerSourceConverters(MapperFactory mapperFactory, ClassMapBuilde
     private class SourceMapper<T, U> extends CustomMapper<SourceAware, SourceAwareEntity<?>> {
 
         @Override
```
Comment (amontenegro) [inline orcid-core/src/main/java/org/orcid/core/adapter/impl/MapperFacadeFactory.java]: Hey @wjrsimpson,

Please take a look at this, I'm thinking that we should map the source from DB to JAXB only, not the other way, and, in the managers, before storing the DB element we manually set the source.

Thanks

Comment (wjrsimpson) [inline orcid-core/src/main/java/org/orcid/core/adapter/impl/MapperFacadeFactory.java]: Yes, sounds good to me.

**Pull request merged**